DOC: CI failure due to fsspec deprecation warning

There is CI failure in the ```Web/Docs```, under ```check ipython directive errors``` on github actions, noticed on #32831 and on #32830 

The error that is being shown:

```
Run ! grep -B1 "^<<<-------------------------------------------------------------------------$" sphinx.log
  FutureWarning,
<<<-------------------------------------------------------------------------
##[error]Process completed with exit code 1.
```

---

The root cause of the problem, is a warning that is generated during the creation of the docs.

```
>>>-------------------------------------------------------------------------
Warning in /home/runner/work/pandas/pandas/doc/source/user_guide/scale.rst at block ending on line 254
 Specify :okwarning: as an option in the ipython:: block to suppress this message
 ----------------------------------------------------------------------------
 /home/runner/miniconda3/envs/pandas-dev/lib/python3.7/site-packages/fsspec/implementations/local.py:33: FutureWarning: The default value of auto_mkdir=True has been deprecated and will be changed to auto_mkdir=False by default in a future release.
   FutureWarning,
<<<-------------------------------------------------------------------------
```

--- a/doc/source/user_guide/scale.rst
+++ b/doc/source/user_guide/scale.rst
@@ -249,7 +249,7 @@ We can use Dask's ``read_parquet`` function, but provide a globstring of files t
 
    import dask.dataframe as dd
 
-   ddf = dd.read_parquet("data/timeseries/ts*.parquet", engine="pyarrow")
+   ddf = dd.read_parquet("data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True)
    ddf
 
 Inspecting the ``ddf`` object, we see a few things

--- a/doc/source/user_guide/scale.rst
+++ b/doc/source/user_guide/scale.rst
@@ -248,6 +248,7 @@ We can use Dask's ``read_parquet`` function, but provide a globstring of files t
 .. ipython:: python
 
    import dask.dataframe as dd
+   :okwarning:
 
    ddf = dd.read_parquet(
        "data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True

--- a/doc/source/user_guide/scale.rst
+++ b/doc/source/user_guide/scale.rst
@@ -249,7 +249,9 @@ We can use Dask's ``read_parquet`` function, but provide a globstring of files t
 
    import dask.dataframe as dd
 
-   ddf = dd.read_parquet("data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True)
+   ddf = dd.read_parquet(
+       "data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True
+   )
    ddf
 
 Inspecting the ``ddf`` object, we see a few things

--- a/doc/source/user_guide/scale.rst
+++ b/doc/source/user_guide/scale.rst
@@ -250,9 +250,7 @@ We can use Dask's ``read_parquet`` function, but provide a globstring of files t
 
    import dask.dataframe as dd
 
-   ddf = dd.read_parquet(
-       "data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True
-   )
+   ddf = dd.read_parquet("data/timeseries/ts*.parquet", engine="pyarrow")
    ddf
 
 Inspecting the ``ddf`` object, we see a few things

--- a/doc/source/user_guide/scale.rst
+++ b/doc/source/user_guide/scale.rst
@@ -246,9 +246,9 @@ We'll import ``dask.dataframe`` and notice that the API feels similar to pandas.
 We can use Dask's ``read_parquet`` function, but provide a globstring of files to read in.
 
 .. ipython:: python
+   :okwarning:
 
    import dask.dataframe as dd
-   :okwarning:
 
    ddf = dd.read_parquet(
        "data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True

